docs/reference/mapping/types/numeric.asciidoc

```diff
@@ -137,7 +139,26 @@ The following parameters are accepted by numeric types:
 
     Accepts a numeric value of the same `type` as the field which is
     substituted for any explicit `null` values.  Defaults to `null`, which
-    means the field is treated as missing.
+    means the field is treated as missing.  Note that this cannot be set
+    if the `script` parameter is used.
+
+`on_script_error`::
+
+    Defines what to do if the script defined by the `script` parameter
```

Comment:
Sorry to be leaving a comment so late -- was just looking how this turned out! I was wondering if we considered re-using the `ignore_malformed` boolean flag here instead of introducing a new parameter? That feels simple and consistent to me, and avoids introducing a new enum parameter.

Reply:
thanks Julie, good question. It all started from "let's support some ignore_malformed behaviour" but then we decided to make it specific to scripts especially as we were not convinced that the two would have the same default values, as far as I recall. Another idea could be to support the new parameter as part of the definition of the script:

```
"field" : {
  "script" : {
    "source" : "here goes the script",
    "on_error" : "continue|fail"
  }
}
```

There is still time to change this if we want to, so let's discuss this.

Reply:
We did discuss re-using `ignore_malformed` but there are some important differences in the semantics of the two parameters.  In particular, `ignore_malformed` is specifically targeted at handling input data that cannot be converted into a number; because the scripts are typed, there is no way that they can return a malformed object, and so the same logic doesn't really apply.  In addition, a script could throw an error for any number of reasons (eg logic errors, array index out of bounds exceptions) and `ignore_malformed` feels too specific to cover all of these.

Reply:
It's nice that we are reusing the `_ignored` meta field though. :+1:

Reply:
Thanks everyone for the context. I agree the name would feel a bit off, maybe if it had been called `ignore_parse_errors` we could have reused it!